main.js

```diff
@@ -0,0 +1,94 @@
+/*
+ Bai 1, So sanh 2 array
+*/
+
+const arr2 = [1, 2, 3, 4]
+const arr3 = [1, 2, 3, 5]
+const arr4 = [1, 2, 3, 4, 5]
+
+const compareArray = (array1, array2) => JSON.stringify(array1) === JSON.stringify(array2)
```

Comment:
Vd:
```javascript
  const arr1 = [1, 2]
  const arr2 = [2, 1]
  
  // sau khi stringify
  const str1 = "[1, 2]"
  const str2 = "[2, 1]"
```
So sánh vẫn phải trả về `arr1` is equal `arr2`

Nếu stringify như bên trên sẽ bị sai

Reply:
Em cảm ơn anh